File: competitiveProgramming/30DaysHR/php/Day2.php
<?php
$_fp = fopen("php://stdin", "r");
$mealCost = fgets($_fp);
$tipPercent = fgets($_fp);
$taxPercent = fgets($_fp);

$totalCost = $mealCost+($mealCost*($tipPercent+$taxPercent)/100);

echo "The total meal cost is ", round($totalCost), " dollars.";
?>

File: competitiveProgramming/30DaysHR/c/Day2.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() 
{
    float mealCost;
    int tipPercent, taxPercent;
    scanf("%f",&mealCost);
    scanf("%d",&tipPercent);
    scanf("%d",&taxPercent);
    
    mealCost += (mealCost*(tipPercent+taxPercent)/100);
    int totalCost = (int)round(mealCost);
    
    printf("The total meal cost is %d dollars.", totalCost);
    return 0;
}
